catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -89,7 +97,7 @@ class MDCNodeListViewController: CBCNodeListViewController {
     appBar.navigationBar.tintColor = UIColor.white
     appBar.navigationBar.titleTextAttributes = [
       NSForegroundColorAttributeName: UIColor.white,
-      NSFontAttributeName: appBarFont! ]
+      NSFontAttributeName: appBarFont ]
```

Comment:
On Xcode 8, this generates a warning:
Expression implicitly coerced from UIFont? to Any.
Can we revert this change and the similar one below to silence the warning?

Reply:
I can't seem to regenerate the warning, on both XCode 8 and 8.3.3.